src/intrinsics/prepack/utils.js

```diff
@@ -89,8 +89,8 @@ export function createAbstract(
     );
     if (locString !== undefined) break;
   }
-  if (!name) {
-    let locVal = new StringValue(realm, locString || "(unknown location)");
+  if (typeof name === "undefined") {
```

Comment:
name !== undefined

Reply:
`name === undefined`? Or was the previous implementation incorrect?

Reply:
Sorry, typo on my part. Yes, name === undefined.